packages/amplify-e2e-tests/src/cleanup-e2e-resources.ts

```diff
@@ -53,6 +53,11 @@ type IamRoleInfo = {
   cbInfo?: CodeBuild.Build;
 };
 
+type RdsInstanceInfo = {
+  name: string;
```

Comment:
can we change this to `identifier`? That is a more common and known term for RDS resources.